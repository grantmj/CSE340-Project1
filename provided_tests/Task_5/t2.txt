TASKS
    1 5

POLY 
    F(a,b) = (a a + b) (a a + b);


EXECUTE
    INPUT z;
    
INPUTS 5