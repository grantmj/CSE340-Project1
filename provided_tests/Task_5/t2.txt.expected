POLY - EXPANDED
	F(a,b) = a^4 + 2 a^2 b + b^2;
